pointmatcher/DataPointsFiltersImpl.h

```diff
@@ -37,6 +37,8 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #define __POINTMATCHER_DATAPOINTSFILTERS_H
 
 #include "PointMatcher.h"
+#include "time.h"
+#include "stdlib.h"
```

Comment:
I think those two includes (time.h and stdlib.h) will cause problem when compile on Windows. If equivalent function are in Boost, it's better. We had to revert back a lot of those to maintain compatibility a while ago. I would prefer to not have to do that again.